### PR TITLE
added mapbox api

### PR DESCRIPTION
**Pull request by vazquezea96**
help with making sure MapBox API shows up.

### CODE REVIEW

**Review by jennifertieu — CHANGES_REQUESTED** [review]

File: app/layout.tsx [hunk 1]
```diff
@@ -5,11 +5,13 @@ import { Inter } from "next/font/google";
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: "PickleConnect",
+  title: "PickleBallConnect",
```
[the remark this answers is not in the document]
Comment (vazquezea96) [reply app/layout.tsx]: fixed

File: app/components/mapbox-map.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+'use client'
+import * as React from "react";
+import mapboxgl from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import { useState } from "react";
+// import the mapbox-gl styles so that the map is displayed correctly
+
+function MapboxMap() {
```
[the remark this answers is not in the document]
Comment (vazquezea96) [reply app/components/mapbox-map.tsx]: fixed

File: .idea/workspace.xml
[the remark this answers is not in the document]
Comment (vazquezea96) [reply .idea/workspace.xml:1]: removed.

File: app/components/mapbox-map.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+'use client'
+import * as React from "react";
+import mapboxgl from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import { useState } from "react";
```
[the remark this answers is not in the document]
Comment (vazquezea96) [reply app/components/mapbox-map.tsx]: changed to import { useState, useRef, useEffect } from 'react';

File: app/components/mapbox-map.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    // save the map object to React.useState
+    setMap(mapboxMap);
+
+    return () => {
+      mapboxMap.remove();
+    };
+  }, []);
+
+  return <div ref={mapNode} style={{ width: "100%", height: "100%" }} />;
```
[the remark this answers is not in the document]
Comment (vazquezea96) [reply app/components/mapbox-map.tsx]: changed to className="h-screen"

**Pull request merged**